frontend/src/plugins/veeValidate/oneEmojiOrTwoCharacters.js

```diff
@@ -0,0 +1,9 @@
+import { size } from 'lodash'
+
+export default (i18n) => ({
+  validate: (value) => {
+    return /\p{Extended_Pictographic}/u.test(value) ? size(value) <= 1 : value.length <= 2
```

Comment:
Wow, I learned something today.
In https://stackoverflow.com/a/72727900 it says that extended pictographic does not match a lone snowflake emoji, is that really the case? Should we use `(\p{Emoji}\uFE0F|\p{Emoji_Presentation})` as that post suggests? Or is that covered in your additional logic with the length?

Reply:
Yes it is covered.